frontend/packages/kubevirt-plugin/src/plugin.ts

```diff
@@ -65,6 +79,26 @@ const plugin: Plugin<ConsumedExtensions> = [
       template: yamlTemplates.getIn([models.VirtualMachineModel, 'default']),
```

Comment:
Shouldn't be yaml template customized as well?

Reply:
Yes, I forgot it :sunglasses: , added.

Reply:
Note, when defining a template we override the `TemplateModel` yaml template, so if we want to support a different yaml for templates we will have to write a new page renderer for `~new`.

Reply:
> Note, when defining a template we override the `TemplateModel` yaml template

YAML templates are keyed by model & template name. In this case, we should do something like:

```ts
{
  type: 'YAMLTemplate',
  properties: {
    model: TemplateModel,
    template: 'VM template YAML goes here',
    templateName: 'vm-template',
  },
},
```

Reply:
Nice, thanks !